.github/workflows/release.yml

```diff
@@ -193,24 +193,53 @@ jobs:
           files: |
             ./tmp/${{ env.artifact_name }}.tgz
 
-#  update-homebrew-formula:
-#    name: Run brew bump-formula-pr for iota on testnet releases
-#    needs: release-build
-#    runs-on: ubuntu-latest
-#    # releasing iota cli on testnet releases because it lags `main` less than mainnet, but is more likely to be stable than devnet
-#    if: ${{ github.event_name == 'release' && contains( inputs.iota_ref, '-rc') }}
-#    steps:
-#      - uses: mislav/bump-homebrew-formula-action@b3327118b2153c82da63fd9cbf58942146ee99f0 # v3.1.0
-#        with:
-#          formula-name: iota
-#          create-pullrequest: true
-#          tag-name: "${{ env.iota_ref }}"
-#          commit-message: |
-#            {{formulaName}} ${{ env.iota_ref }}
-#
-#            Created by https://github.com/mislav/bump-homebrew-formula-action
-#
-#            From release: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
-#        env:
-#          # https://github.com/settings/tokens/new?scopes=public_repo,workflow
-#          COMMITTER_TOKEN: ${{ secrets.HOMEBREW_GH_FORMULA_BUMP }}
+  update-homebrew-formula:
+    name: Update the homebrew-tap formula
+    needs: release-build
+    runs-on: ubuntu-latest
+    # releasing iota cli on testnet releases because it lags `main` less than mainnet, but is more likely to be stable than devnet
+    if: ${{ github.event_name == 'release' && contains(github.ref, '-rc') }}
+    steps:
+      - name: Install Homebrew
+        run: |
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+          echo "/home/linuxbrew/.linuxbrew/bin" >> $GITHUB_PATH
+      - name: Install github CLI
+        run: brew install gh
+      - name: Clean up and validate ref to build and create artifact name
+        shell: bash
+        run: |
+          # Manual dispatch on a branch ref
+          if [ "${{ github.event_name == 'workflow_dispatch' }}" == "true" ]; then
```

Comment:
Am I stupid or does this check make no sense? The action only runs if the if condition of the job is met. But that checks for event_name == release 🤔

Reply:
Good call. I just copied this logic from the build job but you are right that this doesn't make sense